src/utils.js

```diff
@@ -44,6 +44,13 @@
         }
     };
 
+    // Unescape HTML-escaped characters (e.g. transform occurrences of '&amp;' to '&')
+    var unescapeHTML = function(htmlEscapedText) {
```

Comment:
Please put the comment below the function declaration and put a space between `function` and the `parameter list`, I.e. `function (htmlExcapedText) {` as is the convention elsewhere in the code.